src/components/ui/UiLink/UiLink.tsx

```diff
@@ -19,10 +20,18 @@ const theme = createTheme({
   },
 });
 
-function UiLink({ children }: { children: React.ReactNode }) {
+function UiLink({
+  children,
+  props,
+}: {
+  children: React.ReactNode;
+  props: any;
+}) {
   return (
     <ThemeProvider theme={theme}>
-      <Link href="#">{children}</Link>
+      <Link href="#" {...props}>
```

Comment:
UILink is universal component, it shouldn't already have href with specific value, this breaks logic of UILink component